visualizer-server/Cargo.toml

```diff
@@ -19,15 +19,20 @@ anyhow = "1.0"
 async-trait = "0.1"
 bytes = "1.2"
 config = "0.13"
-log = "0.4"
 env_logger = "0.9"
 futures = "0.3"
+log = "0.4"
```

Comment:
Nit: do we actually need one?

Reply:
right, I will replace all `log::` with `tracing::` and will remove this crate